pages/instructorAssessmentQuestionManualGrading/instructorAssessmentQuestionManualGrading.js

```diff
@@ -32,7 +32,11 @@ router.get('/', function (req, res, next) {
       assessment_id: res.locals.assessment.id,
       assessment_question_id: res.locals.assessment_question_id,
     };
-    sqlDb.query(sql.select_instance_questions_manual_grading, params, function (err, result) {
+
+    let query = res.locals.assessment.group_work
+      ? sql.select_instance_questions_manual_grading_groups
+      : sql.select_instance_questions_manual_grading;
```

Comment:
We'd like to be robust to the situation where `group_work` is changed while an assessment is in progress, so some assessment instances are for individual students and some are for groups. To do this, one simple strategy is to LEFT JOIN on both groups and users like this: https://github.com/PrairieLearn/PrairieLearn/blob/b3c07403d2f65bc21eba563b43544fe308b9a00e/pages/instructorAssessmentInstances/instructorAssessmentInstances.sql#L49

Ideally the EJS would also render group or user for each row, rather than using `assessment.group_work` to decide, although I see that `instructorAssessmentInstances.ejs` doesn't do this well: https://github.com/PrairieLearn/PrairieLearn/blob/b3c07403d2f65bc21eba563b43544fe308b9a00e/pages/instructorAssessmentInstances/instructorAssessmentInstances.ejs#L220

It's not necessary to fix this before merging this PR, but it would be good to come back to it quickly in a follow-up PR so we don't forget.

Reply:
Ok, sounds good.

Reply:
Ya,  I totally didn't realize that having both types at once was a thing, because I just assumed that what `instructorAssessmentInstances.ejs` was doing was correct.

Reply:
I think it would be fine to choose the headers based on the value of `assessment.group_work` (which will almost always be correct) but we should just display something sensible in the case of rows of the "wrong" type. For example, a group work assessment could have a column called "Group Name" but then for rows without a group (but with a user_id) could just put the user name in that column.

Reply:
I've made an issue so we don't forget: https://github.com/PrairieLearn/PrairieLearn/issues/5092